tests/functional/aws-node-sdk/test/bucket/getLocation.js

```diff
@@ -70,27 +70,12 @@ describeSkipAWS('GET bucket location ', () => {
             });
         });
 
-        describe('without location configuration', () => {
```

Comment:
the test in put should really be here since it is testing the getLocation api call.  in the test in put, you should take the next step and put an object.  see comment in that file.